airbyte-integrations/connectors/destination-jdbc/src/test/java/io/airbyte/integrations/destination/jdbc/copy/s3/S3StreamCopierFactoryTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.jdbc.copy.s3;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+class S3StreamCopierFactoryTest {
+
+  @Nested
```

Comment:
what's the use of `nested` in this context?

Reply:
just to indicate that these tests are specific to the Config record, rather than the factory itself. might actually make more sense to just move the record into its own file though?